.github/workflows/reusable_packaging_and_deployment.yml

```diff
@@ -40,10 +52,16 @@ jobs:
           CC: ${{ steps.install_cc.outputs.cc }}
           CXX: ${{ steps.install_cc.outputs.cxx }}
         run: |
-          pip3 install "conan==1.59" ${{ fromJSON('["", "gcovr"]')[inputs.with-coverage] }}
+          pip3 install "conan==1.59" yq
           conan user
           conan profile new --detect default
+          conan profile update env.CC="${CC}" default
```

Comment:
This is actually something I would suggest to do in cmake. See https://github.com/dice-group/tentris-cwr-pg/blob/f0cfa64a9bdb128b0dab3a98c2b164176ff9c78e/cmake/conan_cmake.cmake#L23

Reply:
`CONAN_DISABLE_STRICT_MODE=1` this sounds like not so much of a good idea at least from what I could gather from the conan documentation

Reply:
Right, but the part `CXX=${CMAKE_CXX_COMPILER};CC=${CMAKE_C_COMPILER}` does the trick done here via the profile. This also helps that the CMake project simply builds and ensures that CMake and Conan use the same settings. Disabling strict mode was necessary at some point because of some buggy library but should not be necessary anymore.

Reply:
Ok got it, makes sense

Reply:
Looks like its working